ports/mimxrt/mpconfigport.h

```diff
@@ -29,9 +29,13 @@
 // Board specific definitions
 #include "mpconfigboard.h"
 #include "fsl_common.h"
+#include "lib/nxp_driver/sdk/CMSIS/Include/core_cm7.h"
 
 uint32_t trng_random_u32(void);
 
+// Config level
+#define MICROPY_CONFIG_ROM_LEVEL (MICROPY_CONFIG_ROM_LEVEL_FULL_FEATURES)
```

Comment:
with this enabled, there are a lot of config options in this file that can now be removed, because they are the default (at this full features level)

Reply:
actually, if this change is not needed for this PR to build and work, I'd suggest reverting this line (ie don't add it) and do that in a separate PR, which also cleans up the other config options

Reply:
About the config levels: I did that clean-up at a certain stage for the SAMD port, and it's due for the mimxrt port. But it's better to make it separate.

Reply:
Actually this line was added in May 2020 by @jimmo, who made the first minimal port for mimxrt. I cannot tell at first glance the side effects of removing it. So I prefer to leave it as it is now and change it in a separate PR, maybe even raising the level to EXTRA_FEATURES. The mimxrt boards all have plenty of flash.

Reply:
OK